# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1326950158 28800
#      Wed Jan 18 21:15:58 2012 -0800
# Node ID bed72845115f3ee331284b1449edb3200c417cab
# Parent  7912e682aa3046738a64132859050bebeaf51ac8
__finish__.m: Clarify internal documentation.

* __finish__.m: Clarify internal documentation.

diff --git a/scripts/startup/__finish__.m b/scripts/startup/__finish__.m
--- a/scripts/startup/__finish__.m
+++ b/scripts/startup/__finish__.m
@@ -18,23 +18,24 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} __finish__ ()
 ## Undocumented internal function.
 ## @end deftypefn
 
 ## Check for the existence of the function/script, @file{finish}, in the
 ## path or current working directory and execute it.  This function is
-## intended to be excecuted upon a clean exit form Octave.  This is
+## intended to be excecuted upon a clean exit from Octave.  This is
 ## accomplished in the system script @file{startup/octaverc} by use of
-## the built-in function @code{onexit}.
+## the built-in function @code{atexit}.
 
 function __finish__ ()
 
   if (exist ("finish", "file"))
     ## No arg list here since finish might be a script.
     finish;
   endif
 
 endfunction
 
+
 ## No test needed for internal helper function.
 %!assert (1)
